examples/multi-gpu-movielens/01-03b-MultiGPU-Download-Convert-ETL-with-NVTabular-Training-with-TensorFlow.ipynb

```diff
@@ -90,20 +90,24 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "unzipping files: 100%|██████████| 8/8 [00:05<00:00,  1.53files/s]\n"
+      "unzipping files: 100%|██████████| 8/8 [00:04<00:00,  1.66files/s]\n"
      ]
     }
    ],
    "source": [
     "# External dependencies\n",
+    "import os\n",
     "import pathlib\n",
     "\n",
     "import cudf  # cuDF is an implementation of Pandas-like Dataframe on GPU\n",
     "\n",
     "from nvtabular.utils import download_file\n",
     "from sklearn.model_selection import train_test_split\n",
     "\n",
-    "BASE_DIR = \"./data\"\n",
+    "INPUT_DATA_DIR = os.environ.get(\n",
+    "    \"INPUT_DATA_DIR\", \"~/nvt-examples/multigpu-movielens/data/\"\n",
```

Comment:
I don't think this is going to do quite the right thing here - like I think this will create a directory '~' relative to the current working directory. 

Maybe something like this instead?
```suggestion
    "    \"INPUT_DATA_DIR\",  os.path.expanduser(\"~/nvt-examples/multigpu-movielens/data/\")\n",
```

Reply:
(using `pathlib.Path.expanduser()`)

Reply:
ahh cool - sorry I missed that